pre_commit/languages/sbt.py

```diff
@@ -0,0 +1,44 @@
+from __future__ import annotations
+
+from typing import Sequence
+
+from pre_commit.hook import Hook
+from pre_commit.languages import helpers
+
+ENVIRONMENT_DIR = None
+install_environment = helpers.no_install
+health_check = helpers.basic_health_check
+get_default_version = helpers.basic_get_default_version
+
+
+def run_hook(
+        hook: Hook,
+        file_args: Sequence[str],
+        color: bool,
+) -> tuple[int, bytes]:
+    # TODO: Improve impl to connect to run commands via SBT server
+    return run_sbt_hook_via_commandline(hook, file_args, color)
```

Comment:
you can write this as `run = run_sbt_hook_via_commandline` but that would be silly -- there's no reason to have two functions is my point

Reply:
This makes more sense in a later PR. However I may need to change this anyway given [comment](https://github.com/pre-commit/pre-commit/pull/2644/files/163e41875472ff581c14d61ef91ce8f5bc2498f3#r1051139353)